src/plopp/widgets/slice.py

```diff
@@ -88,14 +78,54 @@ def _plopp_observe_(self, callback: Callable, **kwargs):
             self.controls[dim]['slider'].observe(callback, **kwargs)
 
     @property
-    def value(self) -> Dict[str, int]:
+    def value(self) -> Dict[str, int | tuple[int]]:
```

Comment:
From which python version were these supported? I think we are saying Plopp sitll works with python 3.9. Do we need to use `Union` instead?